app/models/request.rb

```diff
@@ -33,6 +33,7 @@ class Request < ApplicationRecord
   enum status: { pending: 0, started: 1, fulfilled: 2, discarded: 3 }, _prefix: true
 
   validates :distribution_id, uniqueness: true, allow_nil: true
+  validate :item_requests_uniqueness_by_item_id
```

Comment:
Right -- this is the one that could be annoying if we didn't do the data cleanup.

Reply:
... but I haven't found any places where we actually do editing of item requests after create. So probably fine actually.

Reply:
I changed it to be a validation on `item_requests`.